.github/workflows/publish.yml

```diff
@@ -8,26 +8,14 @@ on:
 
 jobs:
   publish-artifacts:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
       # Set fetch-depth: 0 to fetch commit history and tags for use in version calculation
       - name: Check out code
-        uses: actions/checkout@v2.3.4
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
-      - name: create checksum file
-        uses: hypertrace/github-actions/checksum@main
-
-      - name: Cache packages
```

Comment:
Correct, the gradle action does the caching now. It delegates to gradle's own action which does a much better job than us. If you look at the build summary, it'll include details about this (no cache yet because only a merged branch can populate it https://github.com/hypertrace/java-grpc-utils/actions/runs/5280504792/attempts/1#summary-14294660225 )

Reply:
Got it.

Reply:
Better example, after merging https://github.com/hypertrace/java-grpc-utils/actions/runs/5281004452/attempts/1#summary-14296270285